src/test/groovy/org/shipkit/internal/gradle/plugin/PluginValidationPluginTest.groovy

```diff
@@ -0,0 +1,12 @@
+package org.shipkit.internal.gradle.plugin
+
+import testutil.PluginSpecification
+
+
+class PluginValidationPluginTest extends PluginSpecification {
+
+    def "apply"() {
+        expect:
+        project.plugins.apply(PluginValidationPlugin)
```

Comment:
We already have quite some "apply" tests which are implemented in this way. Therefore I think this one is ok for now. We can do a bulk change later on.